app/views/events/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <%- model_class = Event -%>
 <div class="page-header">
-  <h1><%=t '.title', :default => model_class.model_name.human.titleize %></h1>
+  <h1><%= @event.kind.humanize %> <%=t '.title', :default => model_class.model_name.human.titleize %></h1>
```

Comment:
This only works because "Workshop" and "Camp" are spelled the same in English and German.